app/utils/schemas/users.js

```diff
@@ -0,0 +1,87 @@
+const {
+  typeError,
+  keyNotExist,
+  invalidEmail,
+  notAlphanumerical,
+  invalidMinLength
+} = require('../../helpers/schema_errors');
+
+const {
+  EMAIL_WOLOX_REGEX,
+  EMAIL_WOLOX_AR,
+  EMAIL_WOLOX_CO,
+  EMAIL_WOLOX_CL,
+  EMAIL_WOLOX_MX,
+  ALPHANUMERICAL_REGEX,
+  PASSWORD_MIN_LENGTH
+} = require('../constants');
+
+exports.signUpSchema = {
+  id: {
```

Comment:
this field is auto-generated by the database

Reply:
Fixed it! :)